pkg/external_repos/redhat_repos.go

```diff
@@ -27,6 +27,7 @@ type RedHatRepo struct {
 	Selector            string `json:"selector"`
 	GpgKey              string `json:"gpg_key"`
 	Label               string `json:"content_label"`
+	FeatureName         string `json:"feature_name"`
```

Comment:
It might be helpful to go ahead and add the ability to update the feature if it changes, i think its just adding it to the list here: https://github.com/content-services/content-sources-backend/blob/main/pkg/dao/repository_configs.go#L1281

Reply:
oh you know, i realized its not actaully importing the feature name into the db.  mind adding that? :)